src/restore.c

```diff
@@ -52,8 +52,8 @@ static bool open_file(const char *file_path, as_vector *ns_vec, io_read_proxy_t
 static bool check_set(char *set, as_vector *set_vec);
 static void * restore_thread_func(void *cont);
 static void * counter_thread_func(void *cont);
-static const char * print_set(const char *set);
-static bool compare_sets(const char *set1, const char *set2);
+static const char * print_optional_str(const char *set);
+static bool compare_strs(const char *set1, const char *set2);
```

Comment:
the parameters are still names `set` and `set2` despite being changed in the definition.